tests/scripts/deploy_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+BASE_DIR=$(cd "$(dirname "$0")"; pwd)
+
+sudo cp /etc/kubernetes/admin.conf $HOME/
```

Comment:
The integration tests are not running in my local test environment. This really needs to be working in both developer environments and the CI. I was able to get it to successfully start in my environment by doing the following:
- Comment out lines 5 and 6. 
   - It is important to remove `sudo` from this script so we don't prompt for the password. We're just running in the context of the logged in user.
- Modify line 7 to pick up my default config: `export KUBECONFIG=~/.kube/config`

What about the following approach to get it to work in both dev environments and the CI?
1. If KUBECONFIG is set as an env var to the tests, use it
1. Else if ~/.kube/config is found, set KUBECONFIG to that path
1. Else set KUBECONFIG to admin.conf as you have already in this script

Reply:
Added this condition on the file